cm/lib/CMReconstruction.hh

```diff
@@ -42,6 +42,9 @@ public:
   /** Save to file */
   void Write(TString filename) const;
 
+  /** Calculate H matrix and save it in file */
+  void HmatrixToFile(TString filename);
```

Comment:
It is more efficient to pass (const) object arguments as references:
    const TString & filename

otherwise object must be fully copied. Reference passes only address (no copy constructor involved) and ```const``` assures that object will not be modified.